tests/test_okf_frontmatter_and_footers.py

```
@@ -0,0 +1,132 @@
+            self.assertTrue(bool(fm_data["type"].strip()), f"{rel_path} 'type' must not be empty")
+
+            self.assertIn("title", fm_data, f"{rel_path} missing 'title'")
+            self.assertIsInstance(fm_data["title"], str, f"{rel_path} 'title' must be a string")
+            self.assertTrue(bool(fm_data["title"].strip()), f"{rel_path} 'title' must not be empty")
+
+            # Assert ISO-8601 parsable timestamp
+            self.assertIn("timestamp", fm_data, f"{rel_path} missing 'timestamp'")
+            ts_str = str(fm_data["timestamp"])
+            try:
+                datetime.fromisoformat(ts_str.replace("Z", "+00:00"))
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)